mrphowl/javascript-track
# JS-108: JavaScript Basics - Introducing Variables

# Related Tickets
[JS-108](https://codeandtheory.atlassian.net/browse/JS-108)
# Description
## Code Challenge: Create a Variable
Create a variable named `myName` and assign my name to it.
## Code Challenge: Update the Value of a Variable
Adjust the code so that the **points** variable holds the expected value.
# Validation
- **Create a Variable**: the variable should be created
  - No error upon assignment of the variable
  - We can add `console.log(myName);` below the declaration to output the value of the new variable.
    ```sh
    > node app.js
    Paul Eliczar Adraneda
    ```
- **Update the Value of a Variable**: the value of the variable should be updated
  - The output should be 150
    ```sh
    > node app.js
    150
    ```
# Risks
None
